packages/babel-plugin-transform-bigint/src/index.js

```diff
@@ -0,0 +1,41 @@
+import syntaxBigInt from "babel-plugin-syntax-bigint";
+
+export default function({ types: t }) {
+  return {
+    inherits: syntaxBigInt,
+
+    visitor: {
+      Program: {
+        exit(path: NodePath, state: Object) {
+          state.file.addImport("bigint-polyfill", "default", "BigInt");
+        },
+      },
+      BinaryExpression(path) {
+        if (path.node.operator === "+") {
+          if (
+            path.node.left.type === "BigIntLiteral" &&
+            path.node.left.type === "BigIntLiteral"
+          ) {
```

Comment:
Sorry if I wasn't 100% clear. I'm thinking in handle semantic cases and error into ```_BigIntAdd(a, b)```. This function follows the implementation of https://tc39.github.io/proposal-bigint/#sec-additive-operators in JS.

@hzoo probably we would like to have the "ifs" there to avoid checks at runtime of literals, for example.